src/main/java/uk/gov/companieshouse/api/accounts/ApplicationConfiguration.java

```diff
@@ -20,14 +19,14 @@
     public MessageDigest getMessageDigest() throws NoSuchAlgorithmException {
         return MessageDigest.getInstance(MessageDigestAlgorithms.SHA_256);
     }
-    
-    @Bean
-    public RestTemplate getRestTemplate() {
-        return new RestTemplate();
-    }
-    
+
     @Bean
     public EnvironmentReader getEnvironmentReader() {
         return new EnvironmentReaderImpl();
     }
+
+    @Bean
+    public RestTemplate getRestTemplate() {
```

Comment:
So just to be clear we are injecting this the application context to use in constructor injection?

Reply:
This has been added for the same reason as we added the **EnvironmentReader bean** so that we can autowire it whenever it is needed. 
FYI. RestTemplate has been added for the TNEP validation, outside this PR.